### PR TITLE
OTC-950: District dropdown under FSP displays only mapped district of the logged in user

### PR DESCRIPTION
**Pull request by hirensoni913**
Filter 'ne' added in name and code, The reason for this change is to exclude any district without a parent. 
https://openimis.atlassian.net/browse/OTC-950

### CODE REVIEW

**Comment by delcroip** [comment]
See CI, NotEqual not insstalled

File: location/schema.py [hunk 1]
```diff
@@ -115,9 +126,13 @@ def resolve_health_facilities_str(self, info, **kwargs):
...
         if region_uuid is not None:
             filters += [Q(location__parent__uuid=region_uuid)]
-        if settings.ROW_SECURITY:
-            dist = UserDistrict.get_user_districts(info.context.user._u)
-            filters += [Q(location__id__in=[l.location_id for l in dist])]
+
+        if (kwargs.get('ignore_location') == False or kwargs.get('ignore_location') is None):
+
+          if settings.ROW_SECURITY:
+              dist = UserDistrict.get_user_districts(info.context.user._u)
```
[the remark this answers is not in the document]
Comment (jdolkowski) [reply location/schema.py:133]: @delcroip I was being consistent with other pieces of code. If you search for **if settings.ROW_SECURITY:** it is implemented like that throughout the whole project.

File: location/gql_queries.py [hunk 2]
```diff
@@ -7,10 +8,13 @@
 from location.apps import LocationConfig
 from location.models import HealthFacilityLegalForm, Location, HealthFacilitySubLevel, HealthFacilityCatchment, \
     HealthFacility, UserDistrict, OfficerVillage
+from core.custom_lookups import *
```
Comment (dragos-dobre) [inline location/gql_queries.py:11]: Do we need to import *?

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: location/gql_queries.py (hunk 2 above)
Comment (jdolkowski) [reply location/gql_queries.py]: no we don't, i removed that

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**